src/Core/Resolvers/QueryExecutor.cs

```diff
@@ -636,6 +645,39 @@ DbResultSet dbResultSet
             return resultProperties;
         }
 
+        /// <summary>
+        /// Reads data into jsonString.
+        /// </summary>
+        /// <param name="dbDataReader">DbDataReader.</param>
+        /// <param name="availableSize">Available buffer.</param>
+        /// <param name="jsonString">jsonString to read into.</param>
+        /// <returns>size of data read in bytes.</returns>
+        public int StreamData(DbDataReader dbDataReader, int availableSize, StringBuilder jsonString)
```

Comment:
Access modifier should be private. Callers only need to use one of the `Get*Result*()` functions.

If you need this for testing, you can reference other parts of the code base which use "MakeInternalsVisibleTo"

Reply:
can also make update param name `jsonString` to better indicate that its value is affected by executing this function. e.g. `resultJsonString`.

Reply:
Fixed.